native/cocos/core/scene-graph/NodeEventProcessor.cpp

```diff
@@ -380,6 +380,29 @@ void NodeEventProcessor::off(const CallbacksInvoker::KeyType &type, CallbackInfo
     }
 }
 
+void NodeEventProcessor::offAll(const CallbacksInvoker::KeyType &type, bool /* useCapture = false*/) {
+    //    bool touchEventExist = std::find(TOUCH_EVENTS.begin(), TOUCH_EVENTS.end(), type) != TOUCH_EVENTS.end();
+    //    bool mouseEventExist = std::find(MOUSE_EVENTS.begin(), MOUSE_EVENTS.end(), type) != MOUSE_EVENTS.end();
+    //    if (touchEventExist || mouseEventExist) {
+    //        offDispatch(type, target, useCapture);
+    //
+    //        if (touchEventExist) {
+    //            if (_touchListener && !checkListeners(_node, TOUCH_EVENTS)) {
+    //                event::EventManager::getInstance()->removeEventListener(_touchListener);
+    //                _touchListener = nullptr;
+    //            }
+    //        } else if (mouseEventExist) {
+    //            if (_mouseListener && !checkListeners(_node, MOUSE_EVENTS)) {
+    //                event::EventManager::getInstance()->removeEventListener(_mouseListener);
+    //                _mouseListener = nullptr;
+    //            }
+    //        }
+    //    } else
```

Comment:
debug code?

Reply:
It isn't debug code.
At the beginning of developing v3.6, we make the whole engine running on c++ without ts logic. 
NodeEventProcessor internally depends on system event (touch, mouse) handling which is not needed now, so we comment them temporarily, maybe we need to uncomment them in the further as more and more logic moved to cpp side.